compiler/damlc/daml-stdlib-src/Prelude.daml

```diff
@@ -25,3 +25,4 @@ import DA.Internal.Template.Functions as X
 #endif
 import DA.Internal.Compatible as X
 import DA.Internal.Assert as X
+import DA.Internal.Desugar as X (Implements (..))
```

Comment:
perhaps even a new `DA.Internal.Interface`?

Reply:
I like the idea of moving the `Implements` typeclass to a new `DA.Internal.Interface`. This will be necessary for it to show up in the stdlib docs.

The other functions/types (`mkMethod`, `Method`, `HasMethod`) should probably stay in `DA.Internal.Desugar`, they're only needed for desugaring.